src/styles/main.scss

```
@@ -1,3 +1,53 @@
-body {
-  margin: 0;
+@import "./utils/variables.scss";
+@import "./utils/mixins.scss";
+@import "./reset.scss";
+
+html {
+  font-family: Arial, sans-serif;
+}
+
+@each $name, $margin-size in $days {
+  .calendar__start-day--#{$name} >:first-child {
+    margin-left: #{$margin-size};
+  }
+}
+
+@for $i from 28 to 31 {
+  .calendar__month-length--#{$i} >:nth-child(n + #{$i + 1}) {
+    display: none;
+  }
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: calc((#{$block-size} + 1px) * 7 + (#{$calendar-padding} * 2));
+  gap: 1px;
+  padding: $calendar-padding;
+  margin: auto;
+
+  &__day {
+    @include square($block-size);
+    background-color: $block-color;
+    border: 1px solid black;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-size: 30px;
+    cursor: pointer;
+    transition-duration: 0.5s;
```

```suggestion
    transition: background-color 0.5s, transform 0.5s;
```
And move 0.5s to the variable